File: sic-api/Entities/Su/SuVerify.cs
using sic_api.Entities.Db;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace sic_api.Entities.Su;

[Table("su_verify")]
public class SuVerify : BaseEntity
{

    [Required]
    [MaxLength(100)]
    [Column("verify_type")]
    public string VerifyType { get; set; } = default!;

    [Required]
    [MaxLength(300)]
    [Column("reference_number")]
    public string ReferenceNumber { get; set; } = default!;

    [Required]
    [MaxLength(300)]
    [Column("token")]
    public string Token { get; set; } = default!;

    [Required]
    [MaxLength(300)]
    [Column("max_retry")]
    public int MaxRetry { get; set; } = 5;

    [Required]
    [Column("retry_count")]
    public int RetryCount { get; set; } = 0;

    [Required]
    [Column("expire_at")]
    public DateTime ExpireAt { get; set; }

    [Required]
    [MaxLength(255)]
    [Column("recipient")]
    public string Recipient { get; set; } = default!;


}

File: sic-spring/sic/src/main/java/com/softinter/sicapi/entity/su/SuVerify.java
package com.softinter.sicapi.entity.su;

import com.softinter.sicapi.entity.base.BaseEntity;
import jakarta.persistence.*;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Entity
@Table(name = "su_verify")
@Data
@EqualsAndHashCode(callSuper = true)
@NoArgsConstructor
public class SuVerify extends BaseEntity {

    @Column(name = "verify_type", nullable = false, length = 100)
    private String verifyType;

    @Column(name = "reference_number", nullable = false, length = 300)
    private String referenceNumber;

    @Column(name = "token", nullable = false, length = 300)
    private String token;

    @Column(name = "max_retry", nullable = false)
    private Integer maxRetry = 5;

    @Column(name = "retry_count", nullable = false)
    private Integer retryCount = 0;

    @Column(name = "expire_at", nullable = false)
    private Instant expireAt;

    @Column(name = "recipient", nullable = false, length = 255)
    private String recipient;
    
}